service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateLandingZoneFederatedIdentityStep.java

```diff
@@ -0,0 +1,168 @@
+package bio.terra.landingzone.stairway.flight.create.resource.step;
+
+import bio.terra.landingzone.library.landingzones.definition.ArmManagers;
+import bio.terra.landingzone.service.landingzone.azure.model.LandingZoneResource;
+import bio.terra.landingzone.stairway.common.model.TargetManagedResourceGroup;
+import bio.terra.landingzone.stairway.flight.utils.FlightUtils;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.stairway.exception.RetryException;
+import com.azure.resourcemanager.msi.fluent.models.FederatedIdentityCredentialInner;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.V1DeleteOptions;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1ServiceAccount;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+
+public class CreateLandingZoneFederatedIdentityStep implements Step {
+  private static final Logger logger =
+      LoggerFactory.getLogger(CreateLandingZoneFederatedIdentityStep.class);
+  public static final String k8sNamespace = "default";
```

Comment:
Wondering if we should make a dedicated `landing-zone` namespace for the KSA plus deployments for LZ setup

Reply:
is there an advantage to that?

Reply:
Just organization, anticipating there will be more LZ k8s resources. I don't feel strongly, `default` is fine.

On a random cluster, here is what is in the `default` namespace out of the box:
```
➜  ~ kubectl get all -n default
NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
service/kubernetes   ClusterIP   10.0.0.1     <none>        443/TCP   16d
```